backend/moonstream/data.py

```diff
@@ -94,8 +94,8 @@ class EthereumTransaction(BaseModel):
     gas: int
     gasPrice: int
     value: int
-    from_address: str = Field(alias="from")
-    to_address: Optional[str] = Field(default=None, alias="to")
+    from_address: str = Field(alias="from_address")
```

Comment:
@Yhtiyar : You don't need to put `alias="from_address"` if you are just using the same name as the member.

Why do you want to remove the alias? The alias allowed us to respect web3 schema for transactions.

Reply:
Because transaction object in frontend has `from_address` instead of `from`